pkgs/top-level/all-packages.nix

```diff
@@ -3494,12 +3494,11 @@ with pkgs;
 
   ised = callPackage ../tools/misc/ised {};
 
-  isl = isl_0_17;
+  isl = isl_0_20;
   isl_0_11 = callPackage ../development/libraries/isl/0.11.1.nix { };
-  isl_0_12 = callPackage ../development/libraries/isl/0.12.2.nix { };
```

Comment:
Can you also remove the files associated with the attribute?

Reply:
Sorry about that! Just removed these files.